### PR TITLE
Molpro: Input and output files (including xml) for Molpro

### PR DESCRIPTION
**Pull request by sjrl**
Started adding Molpro input and output files to be used for testing purposes. I would like to keep adding more methods. For example a standalone HF, standalone MP2, CCSD, etc. 

How should I organize the directory structure for this? Should I choose one for now and it can be changed later?

### CODE REVIEW

File: molpro/water_energy/input.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import qcengine as qcng
```
Comment (dgasmith) [inline molpro/water_energy/input.py]: Can you move this to a `molpro/scripts` or similar, but keep the input JSON example in the `water_energy/gradient` folders?

**Review by dgasmith — COMMENTED** [review]
This is great, thanks. One more minor change and we can pull this in. The XML looks like it has most of the things that we are interested in.

File: molpro/water_energy/input.py (hunk 1 above)
Comment (sjrl) [reply molpro/water_energy/input.py]: So just to clarify, you mean move the input.py file to the `molpro/scripts` folder, but copy the JSON portion into a a file within the `water_energy/gradient` folder?

Comment (dgasmith) [reply molpro/water_energy/input.py]: Exactly!

**Comment by dgasmith** [comment]
Thanks!

**Pull request merged**